Description: Disable Python directives
 Ledger in unstable has disabled Python support due to the
 Python2 removal from the archive.
Author: Jelmer VernooÄ³ <jelmer@debian.org>

---
Last-Update: 2020-01-01

--- ledger2beancount-1.8.orig/tests/directives.beancount
+++ ledger2beancount-1.8/tests/directives.beancount
@@ -11,7 +11,6 @@
 ;   - Unsupported directive `C` skipped
 ;   - Unsupported directive `D` skipped
 ;   - Unsupported directive `N` skipped
-;   - Unsupported directive `eval` skipped
 ;   - The `define` directive is not supported
 ;   - Unsupported directive `value` skipped
 ;   - The `def` directive is not supported
@@ -211,18 +210,6 @@ poptag #foo
 ; Ignore pricing information
 ; N EUR
 
-; python
-;   def print_type(val):
-;     print type(val), val
-;
-; eval print_type(true)
-
-; python
-;     def option_pyfirst(context):
-;         print "In --pyfirst (from %s)" % context
-;
-;     def option_pysecond(context, val):
-;         print "In --pysecond=%s (from %s)" % (val, context)
 2019-04-28 * "Transactions can follow python without empty line"
   Assets:Test                        10.00 EUR
   Equity:Opening-Balance            -10.00 EUR
--- ledger2beancount-1.8.orig/tests/directives.ledger
+++ ledger2beancount-1.8/tests/directives.ledger
@@ -206,18 +206,6 @@ D 1000.00 EUR
 ; Ignore pricing information
 N EUR
 
-python
-  def print_type(val):
-    print type(val), val
-
-eval print_type(true)
-
-python
-    def option_pyfirst(context):
-        print "In --pyfirst (from %s)" % context
-
-    def option_pysecond(context, val):
-        print "In --pysecond=%s (from %s)" % (val, context)
 2019-04-28 * Transactions can follow python without empty line
     Assets:Test                        10.00 EUR
     Equity:Opening-Balance            -10.00 EUR
